docs/guides/directing-traffic-to-your-app.md

```diff
@@ -0,0 +1,375 @@
+# Directing Traffic to Your Application
+
+In this guide, you will learn how to direct external traffic to your Kubernetes applications using the Gateway API and
+NGINX Kubernetes Gateway. Whether you're managing a web application or a REST backend API, you can use NGINX Kubernetes
+Gateway to expose your application outside the cluster.
+
+## Prerequisites
+
+- NGINX Kubernetes Gateway is [installed](/docs/installation.md) on your cluster.
+- [Expose NGINX Kubernetes Gateway](/docs/installation.md#expose-nginx-kubernetes-gateway) and save the public IP
+  address and port of NGINX Kubernetes Gateway into shell variables:
+
+   ```text
+   GW_IP=XXX.YYY.ZZZ.III
+   GW_PORT=<port number>
+   ```
+
+## The Application
+
+The application we are going to use in this guide is a simple coffee application comprised of one Service and two Pods:
+
+![coffee app](/docs/images/direct-all-traffic-app.png)
+
+With this architecture, the coffee application is not accessible outside the cluster. We want to expose this application
+on the hostname `cafe.example.com` so that users outside the cluster can access it.
+
+To do this, we will install NGINX Kubernetes Gateway and create two Gateway API resources: a Gateway and an HTTPRoute.
```

Comment:
Yep... might sound cleaner to have it at the end, but if we don't immediately explain it, it might be good to hyperlink the two object names to the corresponding page on the Kubernetes Gateway API page.